### ISSUE TITLE
Spyder is not running after updating to 4.1.0 from 4.0.1 

### ISSUE THREAD

**Issue by srvmig** [spyder-ide/spyder#11845]
## Problem Description
After updating the the anconda with conda update --all, spyder is not running.

### What steps reproduce the problem?

1. Updated with conda update --all
2. After opening the spyder the Console is showing the error
3. Console is not working

### Paste Traceback/Error Below (if applicable)

```python-traceback

Traceback (most recent call last):
File "C:\Users\ASUS\Anaconda3\lib\site‑packages\spyder\plugins\ipythonconsole\plugin.py", line 1209, in create_kernel_manager_and_kernel_client
kernel_manager.start_kernel(stderr=stderr_handle, **kwargs)
File "C:\Users\ASUS\Anaconda3\lib\site‑packages\jupyter_client\manager.py", line 267, in start_kernel
self.kernel = self._launch_kernel(kernel_cmd, env=env, **kw)
File "C:\Users\ASUS\Anaconda3\lib\site‑packages\jupyter_client\manager.py", line 211, in _launch_kernel
return launch_kernel(kernel_cmd, **kw)
File "C:\Users\ASUS\Anaconda3\lib\site‑packages\jupyter_client\launcher.py", line 135, in launch_kernel
proc = Popen(cmd, **kwargs)
File "C:\Users\ASUS\Anaconda3\lib\subprocess.py", line 800, in __init__
restore_signals, start_new_session)
File "C:\Users\ASUS\Anaconda3\lib\subprocess.py", line 1207, in _execute_child
startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

## Versions

* Spyder version: 4.1.0
* Python version: 3.7.6
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

**Comment by CyanBC** [comment]
I've been having this problem as well. I managed to wrangle a way out of it by specifically installing the newest version of qt from conda-forge.

I did:
conda install qt=5.12.5

I've noticed that I have a few packages, fiona - that are downgrading qt when I install them.

**Comment by UniqASL** [comment]
I have the same problem here, with the following package versions, also on Windows 10:

- Spyder: 4.1.0
- Python: 3.7.6
- Qt: 5.12.5
- PyQt: 5.12.3

**Comment by Brow71189** [comment]
Same problem here. I checked what file it is looking for (added `print(cmd)` in `launcher.py` before the line that fails):
```
['C:\\Users\\Andi\\Miniconda3\\envs\\spyder\\lib\\site‑packages\\spyder\\plugins\\ipythonconsole\\scripts\\conda‑activate.bat',
'C:/Users/Andi/Miniconda3/Scripts/activate',
'C:/Users/Andi/Miniconda3/envs/spyder',
'C:\\Users\\Andi\\Miniconda3\\envs\\spyder\\pythonw.exe',
'C:\\Users\\Andi\\AppData\\Roaming\\jupyter\\runtime\\kernelc37696574dd.json']
```
At least on my machine , that script it is looking for (`conda-activate.bat`) is not there. In fact, the whole folder `scripts` in `ipythonconsole` is missing.

**Comment by Boltasar** [comment]
I am getting the same error when starting from shortcut, but not when starting from anaconda prompt

**Comment by ererkka** [comment]
> I am getting the same error when starting from shortcut, but not when starting from anaconda prompt

Same here. Just upgraded to Spyder 4.1.0

**Comment by django122** [comment]
Same here, with anaconda navigator no error, but start directly --> same error

**Comment by manfred2020** [comment]
after yesterdays trouble with spyder 4.1 (see other issue thread, which is closed now) i cannot get spyder running (only white screen appears (spyder no repsonse) in a python 3.8.2 environment.

i already deinstalled anaconda completely and reinstalled it. doesn't help.

spyder 4.0.1 works with python 3.7.6 environment, but not with 3.8.2 env. (no matter if i start spyder from ananconda gui or via desktop shortcut).

**Comment by manfred2020** [comment]
update 2 hours later: i manually deleted everything about anaconda, python, etc. on my harddrive and installed anaconda again.
created 2 basic environments: python 3.7.6 and 3.8.2. spyder is running in py 3.7 env but still not running in py 3.8 env!

this trouble since yesterday cost my some nerves...

**Comment by ccordoba12** [comment]
Duplicate of #11819

**Comment by ccordoba12** [comment]
This will be fixed in 4.1.1, to be released later today or tomorrow.
